### ISSUE TITLE
Problem with `\\` at end of an ALIASES in the configuration file

### ISSUE THREAD

**Issue by albert-github** [doxygen/doxygen#8144]
When having the following items in the Doxyfile:
```
QUIET=YES
ALIASES +=bs{1}="A\1\\"
ALIASES +=bs1{2}="A\1\\\2Z3"
ALIASES +=bs2{1}="A\1\\\\"
ALIASES +=bs3{2}="A\1\\\2Z3"
ALIASES +=bs4{1}="A\1\\\\\\"
ALIASES +=bs5{2}="A\1\\\2Z3"
ALIASES +=bs6{1}="A\1\\\\\\\\"
ALIASES +=bs7{2}="A\1\\\2Z3"
ALIASES +=as{1}="A\1@\"
ALIASES +=as1{2}="A\1\\\2Z3"
ALIASES +=as2{1}="A\1@\@\"
ALIASES +=as3{2}="A\1\\\2Z3"
ALIASES +=as4{1}="A\1@\@\@\"
ALIASES +=as5{2}="A\1\\\2Z3"
ALIASES +=as6{1}="A\1@\@\@\@\"
ALIASES +=as7{2}="A\1\\\2Z3"
```
when running `doxygen -x` we get the warnings:
```
warning: Missing end quote (") on line 2, file Doxyfile
warning: Missing end quote (") on line 4, file Doxyfile
warning: Missing end quote (") on line 6, file Doxyfile
warning: Missing end quote (") on line 8, file Doxyfile
warning: Missing end quote (") on line 10, file Doxyfile
warning: Missing end quote (") on line 12, file Doxyfile
warning: Missing end quote (") on line 14, file Doxyfile
warning: Missing end quote (") on line 16, file Doxyfile
```
and output:
```
# Difference with default Doxyfile 1.9.0 (cbfc7f9e6a2532f58e736c2049b32bd231efade8)
ALIASES                = "bs{1}=A\1\\" ALIASES" \
                         +=bs1{2}=A\1\\\2Z3 \
                         "bs2{1}=A\1\\\\" ALIASES" \
                         +=bs3{2}=A\1\\\2Z3 \
                         "bs4{1}=A\1\\\\\\" ALIASES" \
                         +=bs5{2}=A\1\\\2Z3 \
                         "bs6{1}=A\1\\\\\\\\" ALIASES" \
                         +=bs7{2}=A\1\\\2Z3 \
                         "as{1}=A\1@\" ALIASES" \
                         +=as1{2}=A\1\\\2Z3 \
                         "as2{1}=A\1@\@\" ALIASES" \
                         +=as3{2}=A\1\\\2Z3 \
                         "as4{1}=A\1@\@\@\" ALIASES" \
                         +=as5{2}=A\1\\\2Z3 \
                         "as6{1}=A\1@\@\@\@\" ALIASES" \
                         +=as7{2}=A\1\\\2Z3
QUIET                  = YES
```

This issue was discovered during some checking on #8141 and looks like to be present a long time (at least since 1.8.1), but also in the current master version (1.9.0 (cbfc7f9e6a2532f58e736c2049b32bd231efade8)). Adding an empty lines in the Doxyfile doesn't really help.
For completeness also a `aa.md` has been included using the defined ALIASES.

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/5465682/example.tar.gz)

Note: problem also exists in doxywizard

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
